source/docs/guides/index.md

```diff
@@ -89,6 +89,12 @@ cards:
             Schedule series of automatic payments.
         icon: creditcard-outline
         link: /docs/guides/recurring_payment
+    -
+        title: Settlement Report
```

Comment:
This needs to be added to the left menu on the Guides page
http://dev.beanstream.com.oe-679.s3-website-us-east-1.amazonaws.com/docs/guides/

Reply:
Nice catch, fixed.